pom.xml

```diff
@@ -111,6 +112,9 @@
                             -javaagent:${settings.localRepository}/org/jacoco/org.jacoco.agent/${jacoco.version}/org.jacoco.agent-${jacoco.version}-runtime.jar=destfile=${session.executionRootDirectory}/target/jacoco.exec
                         </argLine>
                     </argLines>
+                    <jvm>/Library/Java/JavaVirtualMachines/temurin-17.0.9/Contents/Home/bin/java</jvm>
```

Comment:
By leaving this configuration on, you are enforcing the pipeline to run the tests with this java version, please remove these changes, you should pass this configuration as maven parameters while testing.

Reply:
fixed